packages/gasket-plugin-nextjs/generator/jest/test/index.spec.js

```diff
@@ -3,11 +3,17 @@ import { render, screen } from '@testing-library/react';
 import { IndexPage } from '../pages/index';
 import { expect } from '@jest/globals';
 
+// Fix for noisy jsdom error
+// https://github.com/nickcolley/jest-axe/issues/147#issuecomment-758804533
+const { getComputedStyle } = window;
+window.getComputedStyle = (elt) => getComputedStyle(elt);
+
+jest.mock('react-intl');
```

Comment:
I understand this is a preemptive example mock - but does it blow up if `react-intl` is NOT installed for a particular app? If it still works, then I think it's fine to leave here, but add a comment about why it is in the example test for users to follow up on.

Reply:
I was able to test it out and confirm it works.